oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -207,6 +212,10 @@ public FileSystem getFileSystem() {
      */
     @Override
     public OSProcess getProcess(int pid) {
+    	return getProcess(pid, null, null);

```

Comment:
all this really does is pass the `getUserInfo()` call into the next method and require you to null check to figure out that you need to update it.  Just get that info here and pass it so you don't have to null check later.  Better yet, have that info cached by the UserInfo class.

Reply:
Yes, that makes sense.